exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,5 @@
 const name = " Daniel  ";
-
-console.log(message);
+let exerciseOne =
+"My name is " + name+ " and my name is " + nameLength + " characters long";
```

Comment:
nameLength is not defined, you need to declare the nameLength variable first before you can use it on this line. You should use trim() buildin method to remove whitespace on the sides of the name.

Reply:
Sure, you are right, somehow I deleted the right one and type the wrong code while getting frustrated with the failure of the Mandatory coursework